app/controllers/testonly/StubController.scala

```diff
@@ -23,35 +22,85 @@ object TestSetupForm {
   ))
 }
 
-class StubController @Inject()(val apiController: ApiController,val  journeyRepository: JourneyRepository)(implicit val ec: ExecutionContext, implicit val messagesApi: MessagesApi)
-  extends FrontendController
-    with I18nSupport
+object StubHelper {
+  val regexPatternForId = """(?<=lookup-address\/)(.*)(?=/lookup)""".r
+  val getJourneyIDFromURL = (url:String) => regexPatternForId.findFirstIn(url).getOrElse(throw new Exception("id not in url"))
+
+  def changeContinueUrlFromUserInputToStubV2(journeyconfigV2: JourneyConfigV2, id: String): JourneyConfigV2 =
+    journeyconfigV2.copy(
+      options = journeyconfigV2.options.copy(
+        continueUrl = controllers.testonly.routes.StubController.showResultOfJourney(id).url)
+    )
+
+  val defaultJourneyConfigV2JsonAsString: JsValue = Json.parse(
+    """{
+      "version": 2,
+      "options":{
+      "continueUrl": "This will be ignored"
```

Comment:
Would be better to build from the model and store as a constant, then convert to json here